packages/react/src/useTable.tsx

```diff
@@ -148,6 +151,14 @@ export const useTable = <
     [fieldSelectionMap, metadata?.fields, options]
   );
 
+  const sortColumn = useCallback((colName: string, direction: "Ascending" | "Descending" | undefined) => {
```

Comment:
You don't actually need to include the `undefined` in the type. You can just do something like this to let `direction` become an optional instead of a required parameter that passes in an `undefined`:

```ts
const sortColumn = useCallback((colName: string, direction?: "Ascending" | "Descending") // look at the question mark
```

(⬆️ same for places that include the `undefined` in the type)

Also, it seems that we're using the `"Ascending" | "Descending"` type in quite a few places. How about making it a separate type and sharing it in other areas?